src/components/screens/public/ConsentProvider/ConsentProviderController.js

```diff
@@ -0,0 +1,80 @@
+import { VuexModule, Action, Module } from 'vuex-class-modules';
+import createController from '@/controllers/createController';
+import { accountsStore as accountsStoreModule } from '@/store';
+import i18n from '@/locales/i18n';
+
+const OPEN_URL_TIMEOUT_MS = 5000;
+
+@Module({ generateMutationSetters: true })
+class ConsentProviderController extends VuexModule {
+  error = '';
+
+  constructor(props, { accountsStore = accountsStoreModule }) {
+    super(props);
+    this.accountsStore = accountsStore;
+  }
+
+  openRedirectUrl(url) {
+    window.location.href = url;
+    setTimeout(() => {
+      this.error = i18n.t('components.consentProvider.redirectUrl');
```

Comment:
Is this error related to timeout exception? If yes - not obvious

Reply:
yes, error related to timeout.

how to explain by other methods for make it more obvious?

Reply:
`components.consentProvider.redirectTimeoutError` I guess